### PR TITLE
feat: support sync of mbtiles from S3, support deployment to ECS

### PR DESCRIPTION
**Pull request by philvarner**
This PR adds the following two changes to the default mbtileserver:

- performs an `aws s3 sync` with a s3 directory prior to running the server to copy all the mbtiles files
- adds ECS CDK deployment scripts

### CODE REVIEW

File: deployment/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+```
+# CPU value      |   Memory value
+# 256 (.25 vCPU) | 0.5 GB, 1 GB, 2 GB
+# 512 (.5 vCPU)  | 1 GB, 2 GB, 3 GB, 4 GB
+# 1024 (1 vCPU)  | 2 GB, 3 GB, 4 GB, 5 GB, 6 GB, 7 GB, 8 GB
+# 2048 (2 vCPU)  | Between 4 GB and 16 GB in 1-GB increments
+# 4096 (4 vCPU)  | Between 8 GB and 30 GB in 1-GB increments
+```
+
+https://github.com/consbio/mbtileserver/issues/111#issuecomment-863414620
```
Comment (abarciauskas-bgse) [inline deployment/README.md]: Seems redundant to have this link here?

File: deployment/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+4. Deploy
+
+Deployment presents a chicken-and-egg problem.  The deployment creates the ECR repository, but also needs the
+images that have not yet been deployed to it to run the Tasks. We solve this by first doing a deployment that fails
```
Comment (abarciauskas-bgse) [inline deployment/README.md]: nit: `Tasks` probably doesn't need to be capitalized?

File: deployment/README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+**Note**: your local Docker host must be configured with at least 4GB of memory for this to build correctly.
+
+```bash
+$ cd ../  # to mbtileserver root directory
+$ make
+```
+
+6. Update the image version and redeploy
+
+Configure `MBTILESERVER_STACK_IMAGE_VERSION` to be the image version you want to deploy, e.g., the output of your previous make.
```
Comment (abarciauskas-bgse) [inline deployment/README.md]: Nit: I think `e.g.` should be `i.e.`

**Review by abarciauskas-bgse — APPROVED** [review]

File: deployment/README.md (hunk 1 above)
Comment (philvarner) [reply deployment/README.md]: stray paste, removed.

**Pull request merged**